app/tests/pages/view-application.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import ViewApplication from '../../pages/view-application';
```

Comment:
```suggestion
import ViewApplication from 'pages/view-application';
```
No need to add `..` anymore, you can write relative imports starting form the `app` directory now

Reply:
Updated.